src/measure_theory/integral/layercake.lean

```diff
@@ -0,0 +1,258 @@
+/-
+Copyright (c) 2022 Kalle Kytölä. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kalle Kytölä
+-/
+import measure_theory.integral.interval_integral
+import analysis.special_functions.integrals
+
+/-!
+# The layer cake formula, a.k.a. Cavalieri's principle
+
+In this file we prove the layer cake formula using Fubini's theorem and the two most common
+applications of it: a representation of the integral of a nonnegative function and a representation
+of the integral of the p:th power of a nonnegative function in terms of the "complementary
+cumulative distribution functions" associated with to the measure and the function.
+
+## Main definitions
+
+No new definitions.
```

Comment:
The standard structure of file-level docstrings is only indicative. When there is no new definition, just erase this.